tests/models/ensemble/bidaf_ensemble_test.py

```diff
@@ -0,0 +1,105 @@
+# pylint: disable=no-self-use,invalid-name,line-too-long
+import torch
+
+from allennlp.common.testing import ModelTestCase
+from allennlp.data.dataset import Batch
+from allennlp.models.ensemble.bidaf_ensemble import BidafEnsemble, ensemble
+
+class BidafEnsembleTest(ModelTestCase):
+    def setUp(self):
+        super(BidafEnsembleTest, self).setUp()
+        self.set_up_model('tests/fixtures/bidaf/experiment.json', 'tests/fixtures/data/squad.json')
+        self.model.eval()
+
+    def test_ensemble_chooses_most_votes(self):
+        subresults = [
+                {
+                        "span_start_probs": torch.autograd.Variable(torch.FloatTensor([[0.0009347021696157753, 0.0012768324231728911, 0.9999938261222839, 1.4630706573370844e-05]])),
```

Comment:
You could do `from torch.autograd import Variable` above to save some horizontal space on all of these lines.  Also, you could use some toy probabilities for all of these (like `0.1`, `0.8`, etc.), so that this doesn't get so long.  And so that it's easier for a person looking at this test to do the arithmetic in their head and make sure they agree the test is correct.